.github/workflows/deploy-test-pypi.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to TestPyPI
+
+on:
+  push:
+    branches:
+      - 'release*'
```

Comment:
I think this needs a slash? Or a dash?
```suggestion
      - 'release/*'
```

Reply:
I did `release/*.*.*` for consistency with the tag filter.